src/components/AddressResolver/EndpointEntry/EndpointEntry.tsx

```diff
@@ -135,92 +137,136 @@ export const EndpointEntry: FunctionComponent<EndpointEntryProps> = ({
           entityLookup: features.entityLookup?.location,
         },
       });
+
+      showOverlay(<AddResolverConfirmation />);
     } catch (e) {
       onSetAllError(e.message);
     }
     setIsLoading(false);
   };
 
   return (
-    <tr className={className}>
-      <th>
+    <div
+      className={`flex flex-col bg-white rounded-xl shadow-lg w-80 mt-6 pt-12 pb-8 md:even:bg-cerulean-50 ${
+        isEditable ? "h-130" : "h-auto"
+      } md:bg-none md:w-full md:h-auto md:rounded-none md:shadow-none md:mt-0 md:p-4`}
+      onMouseOver={() => setHoverState(true)}
+      onMouseOut={() => setHoverState(false)}
```

Comment:
basic clean up = put the logic adding class at the very end. no need `onMouseOver` if u gg with tw `group-hover`

```suggestion
      className={`flex flex-col bg-white rounded-xl shadow-lg w-80 mt-6 pt-12 pb-8 md:even:bg-cerulean-50 md:bg-none md:w-full md:h-auto md:rounded-none md:shadow-none md:mt-0 md:p-4 group ${
        isEditable ? "h-130" : "h-auto"
      }`}
```

Reply:
done.